### DISCUSSION TITLE
Bug on NPC index cards

### DISCUSSION THREAD

**Discussion by StarkMaximum** [fariapp/fari-app#463] [Q&A]
I don't know how to describe this any better than the way I'm about to, but 9 times out of 10 when I rearrange blocks on an NPC index card, Fari immediately crashes. And I mean, immediately. Like I move it just slightly and the whole site gives me the blank "Oops! Something went wrong!" page, and I have to refresh and hope I manually saved everything I had just did. All I'm doing is using Advanced Mode to move things like Aspects or Stress in Fate up and down on the card to make it look natural but moving them too much causes a crash. I don't know how to describe it any better than that and I have no idea what might be causing it, but it's really frustrating in the middle of a game.

**Comment by RPDeshaies** [comment]
There's most definitely something wrong with the drag and drop features in the app, I'll look into it

**Reply by StarkMaximum** [reply]
Thank you very much, I appreciate it. Fari is the the best VTT for the kinds of games I like to run.
